WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -246,7 +246,9 @@ private async Task StepOutputRegistrationPhaseAsync(CancellationToken cancellati
 					round.CoordinatorScript = GetCoordinatorScriptPreventReuse(round);
 					coinjoin = AddCoordinationFee(round, coinjoin, round.CoordinatorScript);
 
-					coinjoin = await TryAddBlameScriptAsync(round, coinjoin, allReady, round.CoordinatorScript, cancellationToken).ConfigureAwait(false);
+					var highestFeeRateTask = async () => (await Rpc.EstimateSmartFeeAsync(2, EstimateSmartFeeMode.Conservative, simulateIfRegTest: true, cancellationToken).ConfigureAwait(false)).FeeRate;
```

Comment:
This is incorrect because it is using the feerate at the current time instead of the feerate agreed for the round. All calculations must be done with the same feerate otherwise number will never be okay.

Reply:
That fee is never used for calculations. We decide whether to add an extra output or not.